mne/io/reference.py

```diff
@@ -307,29 +308,37 @@ def set_eeg_reference(inst, ref_channels=None, copy=True, verbose=None):
     ----------
     inst : instance of Raw | Epochs | Evoked
         Instance of Raw or Epochs with EEG channels and reference channel(s).
-    ref_channels : list of str | None
+    ref_channels : list of str | str
         The names of the channels to use to construct the reference. To apply
-        an average reference, specify ``None`` here (default). It will be added
-        as an SSP projector, so it is not immediately applied to the data. If
-        an empty list is specified, the data is assumed to already have a
-        proper reference and MNE will not attempt any re-referencing of the
-        data. Defaults to an average reference (None).
+        an average reference, specify ``'average'`` here (default). If an empty
+        list is specified, the data is assumed to already have a proper
+        reference and MNE will not attempt any re-referencing of the data.
+        Defaults to an average reference.
     copy : bool
         Specifies whether the data will be copied (True) or modified in place
         (False). Defaults to True.
+    projection : bool | None
+        If ``ref_channels='average'`` this argument specifies if the average
+        reference should be computed as a projection (True) or not (False). If
+        ``projection=True``, the average reference is added as an SSP projector
+        and is not applied to the data (it can be applied afterwards with the
+        ``apply_proj`` method. If ``projection=False``, the average
+        reference is directly applied to the data. Defaults to None, which
+        means ``projection==True``, but will change to ``projection==False`` in
+        the next release.
```

Comment:
shouldn't this part of docs be the same everywhere (see also the comment about using mne's doc copying)?